app/stores/NetworkStatusStore.js

```diff
@@ -27,14 +27,14 @@ export default class NetworkStatusStore extends Store {
   _startTime = Date.now();
   _startupStage = STARTUP_STAGES.CONNECTING;
 
-  ALLOWED_TIME_DIFFERENCE = 0;
+  ALLOWED_TIME_DIFFERENCE = 15 * 1000000; // 15 seconds
```

Comment:
What unit is this difference measured in? Normally we are dealing with milliseconds, so 15 seconds would be `15 * 1000` right?

Reply:
In this case we are dealing with `microseconds`. You can easily check by changing your machine time in the settings.